examples/asyncio/sanitize/sanitize_examples/sanitize_no_share.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+import os
+
+import pangea.exceptions as pe
+from pangea.asyncio.services import SanitizeAsync
+from pangea.config import PangeaConfig
+from pangea.response import TransferMethod
+from pangea.services.sanitize import SanitizeContent, SanitizeFile, SanitizeShareOutput
+
+# Set this filepath to your own file
+FILEPATH = "./sanitize_examples/ds11.pdf"
+
+
+async def main() -> None:
+    token = os.getenv("PANGEA_SANITIZE_TOKEN")
+    assert token
+
+    domain = os.getenv("PANGEA_DOMAIN")
+    assert domain
+
+    config = PangeaConfig(domain)
+    # Create Sanitize client with its token and its config
+
+    client = SanitizeAsync(token, config)
+    try:
+        # Create Sanitize file information, setting scan and crd providers
+        file_scan = SanitizeFile(scan_provider="crowdstrike", cdr_provider="apryse")
+
+        # Create content sanitization config
+        content = SanitizeContent(
+            url_intel=True,
+            url_intel_provider="crowdstrike",
+            domain_intel=True,
+            domain_intel_provider="crowdstrike",
+            defang=True,
+            defang_threshold=20,
+            remove_interactive=True,
+            remove_attachments=True,
+            redact=True,
+        )
+        # Disable share output
```

Comment:
It is probably good to be explicit here, but if you want to receive the sanitized file over a `dest_url`, you don't have to specify `share_output` at all. So, _maybe_ something like this is more telling?
```
        # Do not enable share output and its folder
        # share_output = SanitizeShareOutput(enabled=True, output_folder="sdk_examples/sanitize/")

```
I realize, however, that there are many other optional parameters with default values here that we set explicitly in the examples.

Reply:
Yes, we have a lot of optional params that we set in examples, in order to show it, and in order to have a much consistent response and not depend on how the user setup its service